localstack/services/s3/s3_listener.py

```diff
@@ -628,6 +633,25 @@ def strip_chunk_signatures(data):
     return data_new
 
 
+def is_bucket_available(bucket_name):
+    body = {'Code': '200'}
+    exists, code = bucket_exists(bucket_name)
+    if not exists:
+        body = {
+            'Error': {
+                'Code': code,
+                'Message': 'The bucket does not exist'
+            }
+        }
+        exists = False
+        code = code
+        xmltodict.unparse(body)
+        return exists, code, body
+
+    xmltodict.unparse(body)
```

Comment:
I think we can remove lines 651 and 648 - calling `xmltodict.unparse(body)` has no side-effects, and we're not using the results returned from these functions.